os/board/rtl8721csm/src/component/os/tizenrt/rtk_blemgr.c

```diff
@@ -0,0 +1,419 @@
+	return ret;
+}
+
+trble_result_e trble_netmgr_operation_write_no_response(struct bledev *dev, trble_operation_handle *handle, trble_data *in_data)
+{
+	trble_result_e ret = TRBLE_FAIL;
+	ret = rtw_ble_client_operation_write_no_response(handle, in_data);
+	return ret;
+}
+
+
+/*** Peripheral(Server) ***/
+trble_result_e trble_netmgr_get_profile_count(struct bledev *dev, uint16_t *count)
+{
+	*count = rtw_ble_server_get_profile_count();
+	return TRBLE_SUCCESS;
+}
+
+trble_result_e trble_netmgr_charact_notify(struct bledev *dev, trble_attr_handle attr_handle, trble_conn_handle con_handle, trble_data *data)
+{
+	trble_result_e ret = TRBLE_SUCCESS;
+	ret = rtw_ble_server_charact_notify(attr_handle, con_handle, data->data, data->length);
+	return ret;
+}
+
+trble_result_e trble_netmgr_attr_set_data(struct bledev *dev, trble_attr_handle attr_handle, trble_data *data)
+{
+	trble_result_e ret = TRBLE_FAIL;
+	ret = rtw_ble_server_att_set_data_ptr(attr_handle, data->data);
+	if (ret != TRBLE_SUCCESS) {
+		return ret;
+	}
+	ret = rtw_ble_server_att_set_length(attr_handle, data->length);
+	return ret;
+}
+
+trble_result_e trble_netmgr_attr_get_data(struct bledev *dev, trble_attr_handle attr_handle, trble_data *data)
+{
+	trble_result_e ret = TRBLE_SUCCESS;
+	data->data = rtw_ble_server_att_get_data_ptr(attr_handle);
+	data->length = rtw_ble_server_att_get_length(attr_handle);
+
+	if (data->data == NULL || data->length == 0) {
+		ret = TRBLE_FAIL;
+	}
+	return ret;
+}
+
+trble_result_e trble_netmgr_attr_reject(struct bledev *dev, trble_attr_handle attr_handle, uint8_t app_errorcode)
+{
+	trble_result_e ret = TRBLE_FAIL;
+	ret = rtw_ble_server_reject(attr_handle, app_errorcode);
+	return ret;
+}
+
+trble_result_e trble_netmgr_get_mac_addr_by_conn_handle(struct bledev *dev, trble_conn_handle con_handle, uint8_t bd_addr[TRBLE_BD_ADDR_MAX_LEN])
+{
+	trble_result_e ret = TRBLE_SUCCESS;
+	uint8_t *mac;
+	
+	mac = rtw_ble_server_get_mac_address_by_conn_handle(con_handle);
+	if (mac == NULL) {
+		return TRBLE_FAIL;
+	}
+	memcpy(bd_addr, mac, TRBLE_BD_ADDR_MAX_LEN);
+
+	return ret;
+}
+
+trble_result_e trble_netmgr_get_conn_handle_by_addr(struct bledev *dev, uint8_t bd_addr[TRBLE_BD_ADDR_MAX_LEN], trble_conn_handle *con_handle)
+{
+	trble_result_e ret = TRBLE_SUCCESS;
+	*con_handle = rtw_ble_server_get_conn_handle_by_address(bd_addr);
+	return ret;
+}
+
+trble_result_e trble_netmgr_set_adv_data(struct bledev *dev, trble_data *data)
+{
+	trble_result_e ret = TRBLE_FAIL;
+	ret = rtw_ble_server_set_adv_data(data->data, data->length);
+	return ret;
+}
+
+trble_result_e trble_netmgr_set_adv_resp(struct bledev *dev, trble_data *data)
+{
+	trble_result_e ret = TRBLE_FAIL;
+	ret = rtw_ble_server_set_adv_name(data->data, data->length);
+	return ret;
+}
+
+trble_result_e trble_netmgr_conn_is_active(struct bledev *dev, trble_conn_handle con_handle, bool *is_active)
+{
+	*is_active = rtw_ble_server_conn_is_active(con_handle);
+	return TRBLE_SUCCESS;

```

Comment:
Currently, this function always returns success, but I think we'd better return based on the result.
How about returning after checking the return of `rtw_ble_server_conn_is_active`?

Reply:
It returns only true or false of connection.
We can't know the function worked well.

Reply:
Yes, It returns true or false.
I mean, if it returns false, it is better to return false from `trble_netmgr_conn_is_active`.
For this, we should change the return type of `trble_netmgr_conn_is_active` as bool, and we don't need the parameter, `is_active`.
Anyway it is just coding style, so just take note :)